spec/models/collection_spec.rb

```diff
@@ -188,21 +244,21 @@
   end
 
   describe "managers" do
-    let!(:user) {FactoryBot.create(:manager)}
-    let!(:collection) {Admin::Collection.new}
+    let!(:user) { FactoryBot.create(:manager) }
+    let!(:collection) { Admin::Collection.new(unit: FactoryBot.create(:unit)) }
```

Comment:
Is the `unit: FactoryBot.create(:unit)` arg necessary? The Admin::Collection factory should be providing that already and if it isn't, we should fix that factory. 

This applies for the editor and depositor test blocks as well.